Core/Inc/main.h

```diff
@@ -29,10 +29,9 @@ extern "C" {
 
 /* Includes ------------------------------------------------------------------*/
 #include "stm32f4xx_hal.h"
-
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
-
+#include "Tasks.h"
```

Comment:
Why is this included here? It doesn't look like you are using it in this file

Reply:
It is needed to include the functions for each of the tasks when I call `osThreadNew`